### DISCUSSION TITLE
renovate-bot don't work in my repository

### DISCUSSION THREAD

**Discussion by liujiayii** [renovatebot/renovate#19536] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

I installed renovate-bot in my repository, but I did not receive the updated pr.
my repository: https://github.com/element-plus-react/element-plus-react/blob/master/renovate.json

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
https://gist.github.com/liujiayii/5d32e4d5ddad474bf2e1adb3d726338d
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by rarkins** [comment, marked as answer]
It's running in your repo: https://github.com/element-plus-react/element-plus-react/issues/273

Most likely you are expecting rangeStrategy=update-lockfile by default, so please check out that documentation

**Reply by liujiayii** [reply]
Thanks for your reply,It's working now.I think the configuration of renovate is too complex, and the [dependabot](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/configuring-dependabot-to-only-access-private-registries) is relatively easy to configure.
